solr/core/src/java/org/apache/solr/handler/StreamHandler.java

```diff
@@ -199,10 +200,16 @@ public void inform(SolrCore core) {
       .withFunctionName("mult", MultiplyEvaluator.class)
       .withFunctionName("sub", SubtractEvaluator.class)
       .withFunctionName("log", NaturalLogEvaluator.class)
+
       // Conditional Stream Evaluators
       .withFunctionName("if", IfThenElseEvaluator.class)
       ;
 
+      // Date evaluators
```

Comment:
I figured using one class (< 170 lines) to implement 11 date functions was preferable to lots of little classes.  Now I know there's an assumption that 1 class = 1 function, I can re-factor to add all the extra classes.